libminifi/src/c2/C2Agent.cpp

```diff
@@ -539,9 +563,9 @@ void C2Agent::handle_describe(const C2ContentResponse &resp) {
   } else if (resp.name == "manifest") {
     auto keys = configuration_->getConfiguredKeys();
```

Comment:
This, unlike the "configuration" branch does not sanitize the keys. Isn't this an issue?
(Also, if this is this way because of a mistake, removing the code duplication between the two should prevent such things happening.)

Reply:
good catch. Will make the necessary corrections here.